html/elements/input.json

```diff
@@ -1022,6 +1022,50 @@
             }
           }
         },
+        "required": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/input#required",
```

Comment:
Personllay, I think as mdn_url, https://developer.mozilla.org/docs/Web/HTML/Element/input#required is better than https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/required